tests/net_hosted/ssl_verify_callback.py

```diff
@@ -0,0 +1,37 @@
+# test ssl verify_callback
+
+import io
+import socket
+import tls
+
+
+def verify_callback(cert, depth):
+    print(type(cert), len(cert) > 100, depth)
```

Comment:
I suggest putting `"verify_callback"` at the start of this print, to distinguish lines in the output.